elements/document/collection/nuxeo-collection-metadata-layout.html

```diff
@@ -29,6 +29,10 @@
       <label>[[i18n('label.dublincore.title')]]</label>
       <div name="title">[[document.properties.dc:title]]</div>
     </div>
+    <div role="widget" hidden$="[[!document.properties.dc:source]]">
```

Comment:
Please don't forget to leave an empty line of text in between widgets.
```suggestion

    <div role="widget" hidden$="[[!document.properties.dc:source]]">
```